docs/api-docs/partner/becoming-a-partner.md

```diff
@@ -3,26 +3,29 @@
 <div class="otp" id="no-index">
 
 ### On this page
+- [Partner Program](#partner-program)
 - [Applying to the Technology Partner Program](#applying-to-the-technology-partner-program)
+- [App Marketplace](#app-marketplace)
 - [Troubleshooting](#troubleshooting)
 - [Resources](#resources)
 </div> 
 
-Our BigCommerce Partner Program includes access to training, sales enablement, and a referral network to get more clients and build better ecommerce experiences.
 
+## Partner program
+Our BigCommerce Partner Program includes access to training, sales enablement, and a referral network to get more clients and build better ecommerce experiences.
 
 The following sections outline the steps required to sell single-click apps in the [BigCommerce App Marketplace](https://www.bigcommerce.com/apps). Doing so requires a partnership application and acceptance will authorize you to provide best-of-breed solutions to the fast-growing online businesses we host. 

```

Comment:
Do these steps apply for free apps as well? If so, we may want to re-write this sentence. 
```suggestion
The following sections outline the steps required to publish single-click apps in the [BigCommerce App Marketplace](https://www.bigcommerce.com/apps). Doing so requires a partnership application and acceptance will authorize you to provide best-of-breed solutions to the fast-growing online businesses we host. 

```